include/TransformationFrontendActionFactory.h

```diff
@@ -12,6 +12,10 @@
 
 using clang::tooling::FrontendActionFactory, clang::FrontendAction;
 
+/// This class is a factory to create new TransformationFrontendActions. The need for this class
+/// arouse since newFrontendActionFactory (https://clang.llvm.org/doxygen/Tooling_8h_source.html)
+/// function does not accept FrontendActions having specific constructor like
+/// TransformationFrontendActions
```

Comment:
We need this class since the default FrontedActionFactory doesn't accept any parameters for FrontendAction.